src/three-idx.ts

```diff
@@ -59,10 +60,12 @@ export class ThreeIDX {
   public docs: Record<string, TileDocument>
   public ceramic: CeramicApi
   protected _v03ID?: string
+  protected _subscriptionSet: SubscriptionSet
```

Comment:
We need to hold on to the handles, or this is for future if we decide to close cleanly?

Reply:
Future, and clarity of maintenance. We do not have to operate on the handles right now, considering docs are not unloaded.

Reply:
Good to merge @oed ?

Reply:
yep ✅